### PR TITLE
Add meta registration for foreach_mul_

### PR DESCRIPTION
**Pull request by mlazos**
Fixes #ISSUE_NUMBER

### CODE REVIEW

**Review by Chillee — APPROVED** [review]

File: torch/_meta_registrations.py [hunk 1]
```diff
@@ -2733,6 +2733,11 @@ def meta__foreach_add__list(self, other, alpha=1):
     _check_foreach_binop_tensor_lists(self, other)
 
 
+@register_meta([aten._foreach_mul_.List])
```
Comment (voznesenskym) [inline torch/_meta_registrations.py:2736]: Can't this be:

```
@register_meta([aten._foreach_div_.List, aten._foreach_mul_.List, aten._foreach_add_.List])
def meta__foreach_inplace_op__list(self, other):
    _check_foreach_binop_tensor_lists(self, other)
```

**Review by voznesenskym — APPROVED** [review]

File: torch/_meta_registrations.py (hunk 1 above)
Comment (mlazos) [reply torch/_meta_registrations.py:2736]: add has an alpha :(

**Pull request closed without merging**